docs/whats-new.asciidoc

```diff
@@ -4,112 +4,140 @@
 
 [role="screenshot"]
-image::whats-new/images/8.2/blocklist-page.png[]
+image::whats-new/images/8.3/detection-response-dashboard.png[]
 
 [discrete]
-== Session View tool shows Linux process executions (beta)
-
-{security-guide}/session-view.html[Session View] is a new tool that shows detailed information about Linux process executions in a chronological and hierarchal context. Use Session View to investigate alerts, user activity, and sessions on your Linux infrastructure.
+== New integrations
 
-[role="screenshot"]
-image::whats-new/images/8.2/session-view.png[]
+Several new https://docs.elastic.co/integrations[integrations] have been added, including ones for CIS Kubernetes Benchmark, AWS Security Hub, Cloudflare, Jamf, and Palo Alto Networks.
 
 [discrete]
-== Deploy DGA and Living-off-the-land supervised models in {fleet}
+== Technical preview features
 
-Incorporating supervised models into integration packages allows you to seamlessly install package artifacts inside {kib} with a single
-click. Now you can deploy
-https://docs.elastic.co/en/integrations/dga[DGA]
-and https://docs.elastic.co/en/integrations/problemchild[Living-off-the-land (LotL)]
-detection packages within Fleet.
+*Cloud Security Posture Management*
 
-[role="screenshot"]
-image::whats-new/images/8.2/dga.png[]
+Cloud Security Posture Management (CSPM) and Kubernetes security posture management (KSPM) help you compare your cloud and Kubernetes infrastructure to security best practices. Refer to our {security-guide}/security-posture-management.html[documentation] for setup instructions.
 
+*User risk score*
 
+https://github.com/elastic/detection-rules/blob/209b40b0a30d87898d75bb2d5dc3f2e068b5f09d/docs/experimental-machine-learning/user-risk-score.md[User risk score] assigns a score to highlight risky users within your environment. It utilizes a transform with a scripted metric aggregation to calculate scores based on detection rule alerts within a 90-day window. The transform runs hourly to update the score as new detection rule alerts are generated. Each user risk score is normalized on a scale of 0 to 100.
 
 [discrete]
-== Wildcard support for event filters
+== New Authentications tab added to Users page
+
+An *Authentications* tab has been added to the Users page to show successful and failed authentication events per user.
+
+[role="screenshot"]
+image::whats-new/images/8.3/user-auth.png[]
 
-{security-guide}/event-filters.html[Event filters] now support using wildcard entries for the `file.path.text` field using the `matches` operator.
 
 [discrete]
 == Detection rules enhancements
 
-[discrete]
-=== Rule execution logs
+*New optional settings for event correlation rules*
+
+{security-guide}/rules-ui-create.html#create-eql-rule[Event correlation rules] now allow you to specify the following EQL fields: *Event category*, *Tiebreaker*, and *Timestamp* fields.
+
+*{ml-cap} rules upgraded to V3 {ml} jobs*
```

Comment:
I started using lowercase `v3` since the ML job names all use lowercase. Not sure if we use code formatting in sub-headings like this:

```suggestion
*{ml-cap} rules upgraded to `v3` {ml} jobs*
```